src/pages/components/content-item.mdx

```diff
@@ -15,104 +13,115 @@ import ResourceLinks from 'components/ResourceLinks';
+| _Pictogram_  | Pictograms add visual interest to content items that help reinforce understanding about a particular topic. Pictograms excel at representing abstract ideas such as industries, technology, and functions. _Pictogram can appear in a horizontally or vertically-aligned content item_.                                                                                                  |
+
+### Orientation
+
+Once a media element is added to a content item, you can select its orientation for certain media elements: either vertical or horizontal. Note that when appearing a group, each content item should have the same orientation.
 
 <Row>
 
-<Column colMd={4} colLg={6}>
+<Column colMd={4} colLg={4}>
 
-![A visual example of the Content item component](../../images/component/content-item/component--content-item--behaviors-desktop.png)
+![Content item with vertically alignment, where the pictogram appears above the heading, copy, and link](../../images/component/content-item/content-item-orientation-I.png)
 
-<Caption>Content item on 8 columns</Caption>
+<Caption>Content item with vertically alignment</Caption>
 
 </Column>
 
 <Column colMd={4} colLg={4}>
 
-![A visual example of the Content item component](../../images/component/content-item/component--content-item--behaviors-mobile.png)
+![Content item with horizontal alignment, where the pictogram appears next to the heading, copy, and link](../../images/component/content-item/content-item-orientation-II.png)
 
-<Caption>Content item on 4 columns</Caption>
+<Caption>Content item with horizontal alignment</Caption>
 
 </Column>
 
 </Row>
 
-## Tips and techniques
+| Variant      | Purpose                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| ------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| _Vertical_   | The vertical orientation places the media addition above the heading and body copy content across all breakpoints. Vertical orientation supports all media additions for content item: Image, Pictogram, App icon, Percentage, and Logos.                                                                                                                                                                                                     |
```

Comment:
Yes! I didn't flesh this out enough.

I clarified in the latest that the `pictogram` version can substitute an app icon at the same size instead of the pictogram.